app/jobs/index_finding_aid_job.rb

```diff
@@ -10,9 +10,26 @@ def perform(path, repo_id)
     stdout_and_stderr, process_status = Open3.capture2e(env, cmd)
 
     if process_status.success?
+      dest_path = "#{DulArclight.finding_aid_data}/xml/#{repo_id}"
```

Comment:
We've been changing all the string concatentation to be `File.join` when we can.

Reply:
Okay